packages/pharos/src/components/switch/pharos-switch.scss

```diff
@@ -0,0 +1,127 @@
+@use '../../utils/scss/mixins';
+
+.input-wrapper {
+  @include mixins.option-wrapper;
+
+  align-items: center;
+  column-gap: var(--pharos-spacing-1-x);
+}
+
+.switch__control {
+  height: 32px;
+  width: 70px;
+  box-sizing: border-box;
+  cursor: pointer;
+  border: 1px solid;
+  border-color: var(--pharos-color-marble-gray-80);
+  border-radius: 2rem;
+  background-color: var(--pharos-color-marble-gray-94);
+  transition:
+    background-color 0.3s ease-out,
+    border-color 0.3s ease-out,
+    box-shadow 0.3s ease-out;
+  position: relative;
+
+  &:hover {
+    border-color: var(--pharos-color-marble-gray-30);
+    box-shadow: 0 0 0 1px var(--pharos-color-marble-gray-30);
+  }
+
+  &::before {
+    position: absolute;
+    content: '';
+    height: 15px;
+    width: 15px;
+    background-color: var(--pharos-color-marble-gray-30);
+    right: calc(100% - var(--pharos-spacing-one-and-a-half-x));
+    top: 50%;
+    transform: translateY(-50%);
+    transition: 0.3s ease-out;
+    border-radius: 50%;
+  }
+
+  &::after {
+    font-weight: bold;
+    line-height: 0;
+    text-indent: 0;
+    height: 1rem;
+    color: var(--pharos-color-marble-gray-30);
+    content: 'OFF';
+    text-align: center;
+    display: block;
+    position: absolute;
+    left: calc(100% - 38px);
+    top: 50%;
+    transition: color 0.3s ease-out;
+  }
+
+  .switch__input:checked + .input-wrapper & {
+    background-color: var(--pharos-color-green-97);
+    border-color: var(--pharos-color-green-base);
+
+    &:hover {
+      box-shadow: 0 0 0 1px var(--pharos-color-green-base);
+    }
+
+    &::before {
+      right: var(--pharos-spacing-one-half-x);
+      background-color: var(--pharos-color-green-base);
+    }
+
+    &::after {
+      color: var(--pharos-color-green-base);
+      left: var(--pharos-spacing-one-half-x);
+      content: 'ON';
+    }
+  }
```

Comment:
I wonder if these feel like an appropriate use of the `--pharos-color-feedback-success` alias token since it is sort of indicating feedback, though I understand it's not necessarily indicating "success"...

Reply:
Hmm good question, here too. I think I'd be more apt to move this to success if we also made the light green background (green 97) a success token? Hypothetically could the success color become a greenish-blue, or whatever, and we'd also want to make sure the light variant of that gets updated as well?

Reply:
Hmm a good point. Because I don't want to rush to that decision you outline there, I'm good with leaving these as-is for the moment!